src/processors/transform.rs

```diff
@@ -0,0 +1,293 @@
+#![feature(test)]
+
+extern crate cgmath;
+//extern crate test;
+
+use self::cgmath::{Quaternion, Vector3, Matrix3, Matrix4};
+
+use ecs::{Join, Component, NullStorage, VecStorage, Entity, RunArg, Processor};
+use context::Context;
+use std::sync::{Mutex, Arc};
+use std::collections::{HashMap, VecDeque};
+
+/// Local position and rotation from parent.
+#[derive(Debug, Clone, Copy)]
+pub struct LocalTransform {
+    pub pos: [f32; 3], // translation vector

```

Comment:
Ah whoops, thanks for that catch. Forgot to take those out when I was adding the getter/setters.